### DISCUSSION TITLE
Wildcard Certificates and Multiple AWS Identities

### DISCUSSION THREAD

**Discussion by mauriciocirelli** [polymind-inc/acmebot#504] [Q&A] [answered]
Dear,

We have 3 Route53 DNS zones in AWS and we use 3 different IAM users to manage each of them.
However, looking into the docs, I see that we have only one IAM settings for Route53. In our case, should we deploy 3 instances of ACMEBot Functions?

Also, the examples show how to get certificates for subdomains. Is it possible to request a Wildcard certificate and store it in the respective Key Vault?

Thank you for sharing this project!

**Comment by shibayan** [comment, marked as answer]
Acmebot does not support multiple IAM Users, so it is recommended to use IAM permissions or deploy Acmebot with as many IAM Users as possible.

Wildcard certificates can be obtained. Please give it a try.

**Reply by mauriciocirelli** [reply]
Great, thank you!

One more question: does it support http-01 challenges or just dns-01?

**Reply by shibayan** [reply]
Only dns-01 is supported; there are no plans to support http-01.
